### PR TITLE
Updated readme with possible existing libraries

### LINKED ISSUE #3: Look at existing libraries
### Purpose
To look at existing ROS libraries that could be used.

### Changes

Possibly use existing ROS libraries to improve on current libraries.

### Resources

- https://github.com/ros2/common_interfaces/tree/humble
- https://github.com/UBCSailbot/custom_interfaces/pull/2#issuecomment-1577866615

### PR DESCRIPTION
**Pull request by colinli02**
### Description

- Resolves #3

- Resolves part of issue discussed. Likely to add more onto this PR.

### CODE REVIEW

**Comment by colinli02** [comment]
Note another PR in local_pathfinding:
https://github.com/UBCSailbot/local_pathfinding/pull/22

File: README.md [hunk 1]
```diff
@@ -31,3 +31,19 @@ Used to standardize conventions across external interfaces.
 | HelperHeading       | DesiredHeading, GPS, HelperAISShip |
 | HelperLatLon        | GlobalPath, GPS, HelperAISShip     |
 | HelperSpeed         | GPS, HelperAISShip, WindSensor     |
+
+## Existing Libraries
```
Comment (patrick-5546) [inline README.md]: Can you link to the issue where we discuss this? There's some things in there that I could see being a useful reference in the future

File: README.md [hunk 2]
```diff
@@ -31,3 +31,19 @@ Used to standardize conventions across external interfaces.
...
+
+Possible to-add existing ROS libraries based on ros2 repository:
+[common_interfaces](https://github.com/ros2/common_interfaces) branch ```humble```.
```
Comment (patrick-5546) [inline README.md]: Only need 1 backtick for inline code. Triple backtick is needed for code blocks

File: README.md [hunk 3]
```diff
@@ -31,3 +31,18 @@ Used to standardize conventions across external interfaces.
...
+These can serve as either useful references or can be added entirely.
+
+| ROS Message         | Possible Usage                     |
+| ------------------- | ---------------------------------- |
+| diagnostic_msgs     | Could be used for website sensors  |
+| geometry_msgs       | Simulator, Local Pathfinding       |
+| sensor_msgs         | CAN transceiver reference          |
+| std_msgs            | Possible reference                 |
+| std_srvs            | Possible reference                 |
+| visualization_msgs  | Possible reference          |
```
Comment (patrick-5546) [inline README.md]: Formatting messed up

File: README.md (hunk 1 above)
Comment (colinli02) [reply README.md]: https://github.com/UBCSailbot/custom_interfaces/issues/3#issuecomment-1626875658

File: README.md (hunk 3 above)
Comment (colinli02) [reply README.md]: Fixed, whoops

File: README.md (hunk 2 above)
Comment (colinli02) [reply README.md]: TIL. Thanks

File: README.md (hunk 1 above)
Comment (patrick-5546) [reply README.md]: I meant in the README lol

Comment (colinli02) [reply README.md]: 🤦‍♂️ oops

**Review by patrick-5546 — APPROVED** [review]

**Pull request merged**